views/layouts/application.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Should we add a class layout for styling layouts?
```suggestion
<html class="layout-default">
```

Reply:
@hoangmirs Can I update this change when i'm working on this issue https://github.com/Lahphim/go-crawler-challenge/issues/5?
Because it's about setting up frontend stuff ✌️